gwtp-core/gwtp-dispatch-rest/src/main/java/com/gwtplatform/dispatch/rest/client/core/DefaultResponseDeserializer.java

```diff
@@ -58,7 +59,7 @@ protected DefaultResponseDeserializer(
      * @return <code>true</code> if <code>resultType</code> can be deserialized, <code>false</code> otherwise.
      */
     protected Serialization findSerialization(String resultType, ContentType contentType) {
-        for (Serialization serialization : serializations) {
+        for (Serialization serialization : serializationsProvider.get()) {
```

Comment:
Is this method called often? Maybe you could extract you provider call to a lazy field?